### DISCUSSION TITLE
user login/out audit

### DISCUSSION THREAD

**Discussion by fontexD** [drakkan/sftpgo#1555] [General]
im using the lastet version 2.5.6 but cant seem to find the search/export user audit ? 
we are considering to buy this product and use in prod, but we really need a good logging feature for user login/out 

the combined log json is holding to much info :) 

im referring to this "WebAdmin: allow to search and export audit logs."

**Comment by drakkan** [comment]
You have to install and configure the eventstore/search plugins to have audit logs in WebAdmin. 

If you are truly interested in supporting the project and obtaining support you can contact us directly via email to establish a channel of mutual access and trust. Thank you

**Reply by fontexD** [reply]
awesome, ive installed it and configured it with my db and did the migrate thing and it created the tables, but nothing is beeing logged ? what am i missing out ? 

as soon as we know if this is a viaable solution for us to run in production we will buy support for it for sure

**Reply by fontexD** [reply]
![log](https://github.com/drakkan/sftpgo/assets/87015443/91b2c02a-52d6-4227-854b-de66ca82cfea)

**Reply by nullvalue0** [reply]
> You have to install and configure the eventstore/search plugins to have audit logs in WebAdmin.

Hello, I was wondering if this was this resolved? I am seeing a similar issue. I installed the eventstore/eventsearch plugins and got them working. I am logging to a postgres database. 

- Filesystem events (upload/download/delete) are being logged

- Provider events (new user/group) are also being logged

- However user login/failures are not being logged -- despite following the example sftpgo.json which seems to specify log_events 1, 2, 3 & 4 (login ok, login fail, logout, pw change?).

I have checked the tables directly. eventstore_fs_events & eventstore_provider_events are populating, but eventstore_log_events remains empty.

I'm so close - what could I be missing?

I am using 2.6.6 and the latest plugin releases (1.0.20) - Thanks!

**Comment by fontexD** [comment]
can u give me your email regarding a pro plan

**Reply by drakkan** [reply]
https://sftpgo.com/#services

**Comment by manaschandanroy** [comment]
hi @fontexD  @drakkan  just wanted to ask what actions will insert values in eventstore_provider_events table . notifier shows add , update , delete . so is it file , or user . where does the log will show  if somebody logins
